controllers/external/util.go

```diff
@@ -41,7 +42,7 @@ func Get(c client.Client, ref *corev1.ObjectReference, namespace string) (*unstr
 }
 
 // CloneTemplate uses the client and the reference to create a new object from the template.
-func CloneTemplate(c client.Client, ref *corev1.ObjectReference, namespace string) (*unstructured.Unstructured, error) {
+func CloneTemplate(c client.Client, ref *corev1.ObjectReference, namespace string, owner *metav1.OwnerReference) (*unstructured.Unstructured, error) {
```

Comment:
I think to avoid API breakage in 0.2.x, we should probably create a new function `CloneTemplateWithOwner` and have this one call it with a nil owner. And master already supports an owner, so this would be a release-0.2 only change. @vincepri WDYT?

Reply:
Happy to take that route instead of the varargs approach if it's better for backwards compatibility — just let me know!